### ISSUE TITLE
[Ch 05-06][Notebook][Minor] 셋업 셀에 라이브러리 버전 출력이 없어 수치 드리프트 원인 확정 불가

### ISSUE THREAD

**Issue by modesta-jihye-lee** [yoon-gu/neuqes-101#211]
## 대상

- **챕터**: Ch 05 · Ch 06 (sklearn 구간, `05_sklearn_multiclass` / `06_sklearn_multilabel`)
- **산출물**: Notebook — 첫 셋업 셀(`!pip install -q datasets scikit-learn pandas matplotlib`)
- **기준**: 사용자가 2026-09-01 Colab에서 직접 실행한 두 노트북을 `executed/`와 셀 단위 대조

> **한 이슈로 묶은 이유**: 두 챕터가 같은 유형(셋업 셀에 라이브러리 버전 출력 누락)·같은 수정 경로(한 줄 추가)입니다. `review-drafts/cross/03-nb-minor-library-range.md`(Ch01·02) 선례와 동일한 방식으로 묶습니다.

## 실제 결과

Ch 05·06 두 노트북 모두 `sklearn.__version__`(또는 `datasets.__version__`)을 출력하는 셀이 없습니다. Ch 07·08에는 `torch.__version__` 출력이 있는 것과 대비됩니다.

이 때문에 사용자가 2026-09-01 Colab에서 직접 실행해 `executed/`와 대조한 결과, 두 챕터 모두 소수점 3~4자리대에서 수치가 갈렸는데(아래 표) **원인이 sklearn 버전 드리프트인지 확정할 근거가 없습니다.**

| 챕터 | 항목 | `executed/` | Colab 재실행(2026-09-01) |
|---|---|---|---|
| Ch 05 | Test accuracy (multinomial) | 0.5110 | 0.5090 |
| Ch 05 | OvR accuracy | 0.5070 | 0.5070 (동일) |
| Ch 06 | Subset accuracy | 0.4930 | 0.4920 |
| Ch 06 | Hamming loss | 0.1372 | 0.1376 |
| Ch 06 | micro F1 | 0.7749 | 0.7744 |

두 챕터 모두 계산 관계는 보존됩니다(Ch05 `Improvement = accuracy - 0.2` 양쪽 다 성립, Ch06 threshold 스윕 표의 상대적 순위 동일). 특징이 Ch 01·02에서 이미 확인된 "`TfidfVectorizer(max_features=10000)` 어휘 컷 경계가 sklearn 버전에 따라 갈림" 패턴과 일치해 **환경 의존(sklearn 버전) 드리프트로 추정**하지만, 버전 출력이 없어 검수자가 매번 재현·재계산 없이는 확인할 수 없습니다.

## 기대 결과

두 챕터의 셋업 셀에 한 줄 추가:

```python
import sklearn, datasets
print(f"scikit-learn: {sklearn.__version__}, datasets: {datasets.__version__}")
```

CLAUDE.md "챕터 표준 구조" §1(이슈 #63 합의로 추가된 조항)이 모든 챕터 첫 셀에 라이브러리 버전 출력을 요구합니다 — Ch 05·06은 이 조항이 반영되기 전에 작성된 것으로 보입니다.

## 재현 절차

```bash
grep -c "__version__" 05_sklearn_multiclass/05_sklearn_multiclass.ipynb 06_sklearn_multilabel/06_sklearn_multilabel.ipynb
# 둘 다 0
```

## 심각도

**Minor** — 수치 드리프트 자체는 계산 오류가 아니라 환경 의존(추정)이라 별도 결함이 아닙니다. 버전 출력 누락은 CLAUDE.md 표준 구조 요구사항 위반이자, 이번처럼 향후 검수자가 드리프트 원인을 확정하지 못하게 만듭니다.

## 연결

챕터 검수 이슈 #29(Ch05) · #30(Ch06). 근거: `review-drafts/issue63_comment.md`(Ch01·02 동일 패턴), `review-drafts/cross/03-nb-minor-library-range.md`(같은 방식으로 묶은 선례).
